src/components/ThumbnailsPanel/ThumbnailsPanel.js

```diff
@@ -265,22 +267,40 @@ class ThumbnailsPanel extends React.PureComponent {
 
   render() {
     const { isDisabled, totalPages, display } = this.props;
-
     if (isDisabled) {
       return null;
     }
 
     return (
-      <div className="Panel ThumbnailsPanel" style={{ display }} data-element="thumbnailsPanel" ref={this.thumbnailsPanel}>
-        <div className="thumbs">
-          <ReactList
-            key="panel"
-            itemRenderer={this.renderThumbnails}
-            length={totalPages / this.state.numberOfColumns}
-            type="uniform"
-            useStaticSize
-          />
-        </div>
+      <div
+        className="Panel ThumbnailsPanel"
+        style={{ display }}
+        data-element="thumbnailsPanel"
+        ref={this.thumbnailsPanel}
+      >
+        <Measure
+          bounds
+          onResize={({ bounds }) => {
+            this.setState({
+              height: bounds.height,
+              width: bounds.width,
+            });
+          }}
+        >
+          {({ measureRef }) => (
+            <div ref={measureRef} className="virtualized-thumbnails-container" >
+              <List
+                height={this.state.height}
+                width={this.state.width}
+                rowHeight={180}
```

Comment:
Let's create a constant variable for this number, as the number `180` is used twice in this component. It'd be a bit easier for us to update the height if we change it later on.